Makefile

```diff
@@ -29,6 +29,12 @@ ifeq ($(LANGUAGE),node)
 	FLAGS+= --js_out=import_style=commonjs,binary:$(OUTPUT)
 	FLAGS+= --ts_out=service=grpc-node:$(OUTPUT)
 	FLAGS+= --grpc_out=$(OUTPUT)
+else ifeq ($(LANGUAGE),go)
+	ifeq ($(OUTPUT), $(JAVA_DIR))
+		OUTPUT = $(GO_DIR)
+	endif
+	FLAGS+= --$(LANGUAGE)_out=plugins=grpc:$(OUTPUT)
```

Comment:
Linter würde sagen “value of LANGUAGE is always ‘go’”

Reply:
Isn't that in LANGUAGE=go condition? Should be fine.